Analytics.Portable/Analytics.Portable.csproj

```diff
@@ -41,15 +41,5 @@
   <ItemGroup>
     <PackageReference Include="Newtonsoft.Json" Version="10.0.3" />
   </ItemGroup>
-  <ItemGroup>
-    <AssemblyAttribute Include="System.Runtime.CompilerServices.InternalsVisibleTo">
-      <_Parameter1>Test.UniversalApp</_Parameter1>
```

Comment:
Why do we remove these test projects?

Reply:
These lines were to make visible the internal classes to the Test.UniversalApp project, but the PCL (portable class library) project doesn't support this configuration. I had to add it programmatically in the file _AssemblyConfiguration.cs. But at that moment I forgot to remove these lines. By the way, PCL projects are considered [deprecated](https://docs.microsoft.com/en-us/xamarin/cross-platform/app-fundamentals/pcl?tabs=macos).